.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
You can use global .gitignore file here. 
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Set up 😄